symphony/versions.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "../versions.schema.json",
+  "chain_name": "symphony",
+  "versions": [
+    {
+      "name": "v1.0.1",
```

Comment:
It's quite odd (misleading) to have a version name of v1.0.1, but not have that listed as a compatible version...